src/components/dashboard/tests/Dashboard.test.jsx

```diff
@@ -77,6 +79,17 @@ const defaultCourseState = {
   },
 };
 
+const defaultSubsidyRequestState = {
+  subsidyRequestConfiguration: null,
+  licenseRequests: [],
+  couponCodeRequests: [],
```

Comment:
nit: are these arrays duplicated with the arrays in `requestsBySubsidyType`? Do we need to include both of these?

Reply:
nope, removed!

Reply:
@long74100 should these properties be removed from this object here, too?